microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -19,6 +19,32 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 To submit suggestions and feedback, enter an [Issue about the Add-ons API](https://github.com/MicrosoftDocs/edge-developer/issues/new?title=[Add-ons%20API]).
 
 
+<!-- ------------------------------ -->
+#### Versions of the REST API
+
+As of September 6, 2024, both v2 and v1 of this REST API are supported.  Later, v1 will no longer be supported.  The date for ending v1 support is not yet finalized.
+
+
+<!-- ---------- -->
+###### Important changes in v2
+
+* v2 of this REST API uses API keys, whereas v1 used Secrets instead.
+
+* v2 of this REST API doesn't require sending an Access Token URL.
+
+
+<!-- ====================================================================== -->
+## Getting started with v2 of this REST API
+
+To use v2 of this REST API:
+
+1. At Partner Center, opt-in to the new API key management experience.
+
+1. Re-generate your ClientId and secrets.<!-- todo: reword, b/c secrets are no longer used, according to "v2 of this REST API uses API keys, whereas v1 used Secrets instead" -->  You might need to update your authentication workflows.
```

Comment:
reword "secrets" in this v2 section, b/c secrets are no longer used, according to "v2 of this REST API uses API keys, whereas v1 used Secrets instead"

Reply:
implemented tabsets to completely separate v1.1 info/steps vs v1